docs/getting-started.md

```diff
@@ -23,7 +23,7 @@ container.
 
 ```sh
 $ docker run -v $(pwd):/home/gradle sourcegraph/scip-java:latest scip-java index
-$ src scip upload # (optional) upload index to Sourcegraph
+$ src code-intel upload # (optional) upload index to Sourcegraph
```

Comment:
This repo was originally named "lsif-java" and the invalid `src scip` must be from a bad search/replace